src/components/shoppingCart/cartdetail/CartDetail.jsx

```diff
@@ -3,11 +3,87 @@ import Container from "react-bootstrap/Container";
 import Row from "react-bootstrap/Row";
 import Col from "react-bootstrap/Col";
 import Button from "react-bootstrap/Button";
-import { MOCK_DATABASE } from "../../common/MockDatabase";
+import "./CartDetail.scss";
+import {
+  deleteItemFromCartAction,
+  modifyProductQuantity,
+  sumTotalPrice,
+  addOneMoreItem,
+  removeOneMoreItem,
+} from "../../redux/actions";
+import { useDispatch, useSelector } from "react-redux";
 
-const { title, images, brand, price } = MOCK_DATABASE[0];
+const CartDetail = (props) => {
+  const dispatch = useDispatch();
+  const productsData = useSelector((state) => state.addOrDeleteProductData);
+  // Getting all the total prices from individual items and add them together
+  const itemsTotalPricesArray = useSelector((state) => state.getTotalPrice);
+  const allItemsTotalPrice = itemsTotalPricesArray.reduce((cur, acc) => {
+    return cur + acc;
+  }, 0);
+
+  // Get total price from one item
+  const getItemTotalPrice = () => {
+    const sumTotal = props.info.price * props.info.quantity;
```

Comment:
i feel like this calculation is supposed to be in the reducer, not in the component